src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -32,6 +32,10 @@
                 "title": "Sequential Test Runs",
                 "description": "Activate to first run structural and then behavior test. This feature can help students to better concentrate on the immediate challenge at hand. If activated, you will be provided with more information in a readme file within the test repository. ATTENTION: Can not be changed after creation!"
             },
+            "automaticSubmissionRunAfterDueDate": {
+                "title": "Automatic Submission Run After Due Date",
```

Comment:
I would call this different. We do not really "run a submission", we rather build and test all students submissions.

Reply:
Ok, I have renamed it to `buildAndTestStudentSubmissionsAfterDueDate`.